nautobot/dcim/templates/dcim/devicetype.html

```diff
@@ -71,6 +71,14 @@
                             {{ object.model }}<br/>
                         </td>
                     </tr>
+                    {% if object.hardware_family %}
```

Comment:
I think this field should always be visible for device type and show a placeholder if the field is null.